src/main/java/com/ghilly/service/CountryService.java

```diff
@@ -14,4 +14,6 @@ public interface CountryService {
     void update(Country country);
 
     void delete(int countryId);
+
+    void exists (int countryId);
```

Comment:
Why is it public? You use the method only in CountryServiceRest

Reply:
Done